src/IFedoraApi.php

```diff
@@ -134,4 +134,31 @@ public function saveGraph(
         $uri = "",
         $headers = []
     );
+
+    /**
+     * Creates a version of the resource in Fedora.
+     *
+     * @param string    $uri            Resource URI
+     * @param string    $timestamp      Memento Timestamp
```

Comment:
The Memento spec requires that the [Accept-Datetime and Memento-Datetime](http://mementoweb.org/guide/rfc/#Accept-Memento-Datetime) are RFC-1123 formatted datetimes.

I am 👍  with **not** enforcing that in Chullo and leave it up to the client to make sure their date is correctly formatted, but it might be good to make note of the RFC-1123 formatting in the doc comments here and in FedoraApi.php

Reply:
updated doc comment in 1b26409